catalog-v001.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+    <uri id="User Entered Import Resolution" name="http://sweetontology.net/sweetAll" uri="sweetAll.ttl"/>
```

Comment:
Tangential?

Reply:
Good catch.  Protege doesn't seem to mind if both statements are included, but I swapped out this statement for the one specific to sweet_v23Comments.ttl anyway.